pmg/tasks.py

```diff
@@ -56,8 +56,46 @@ def schedule():
             id="sync-soundcloud",
             replace_existing=True,
             coalesce=True,
-            minute="*/" + app.config["SOUNDCLOUD_PERIOD_MINUTES"],
+            minute="*/" + current_app.config["SOUNDCLOUD_PERIOD_MINUTES"],
         ),
     ]
     for job in jobs:
         log.info("Scheduled task: %s" % job)
+
+
+class StartScheduler(Command):
+    """
+    Start APScheduler and queue scheduled tasks.
+
+    Run with ``python app.py start_scheduler``.
+    """
+
+    def run(self):
+        if not current_app.config["RUN_PERIODIC_TASKS"]:
+            log.info(
+                "Not starting task scheduler because RUN_PERIODIC_TASKS is not 'true'"
+            )
+            return
+
+        from apscheduler.schedulers.blocking import BlockingScheduler
+        from apscheduler.jobstores.sqlalchemy import SQLAlchemyJobStore
+        import pmg.tasks
+
+        scheduler = BlockingScheduler(
```

Comment:
[Docs](https://apscheduler.readthedocs.io/en/stable/modules/schedulers/blocking.html?highlight=BlockingScheduler#apscheduler.schedulers.blocking.BlockingScheduler) for `BlockingScheduler`.